examples/arithmetic/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "uniffi-example-arithmetic"
 edition = "2018"
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
So this looks like it's going to push 0.6.1 to the 0.6.0 branch.  If that's really what we want (I suspect not), perhaps the branch should use the string 0.6.x rather than 0.6.0 in its name.

Reply:
Yesh, I think you're right, `release-v0.6.x` is closer to what I imagine us using here.

Reply:
So what's the right way forward here? Seems like we might be able to (theoretically) change in flight, but given the pre-existing 0.6.{2+}, I'm guessing that might not be practical this time around.  I'm happy to follow your lead here.  I'm going to mark this approved, so that if you're confident you've got something we should go forward with, it won't be blocked from landing.  That said, feel free to ask for another look from me, which I can to Friday Pacific time.

Reply:
Thanks. I don't think the branching setup matters *too* much here this time around, since it's mostly the tags that are important, and they only associate with a commit rather than with a specific branch. What I propose to do here is:

* Update the docs to discuss a `release-v0.6.x`-format branch approach for future reference.
* Merge this to `main` with your r+.
* Push it to a `release-v0.6.x` and see if I can delete this `release-v0.6.0` branch, for completeness.